.github/workflows/pull-request-ci.yml

```diff
@@ -8,7 +8,6 @@ on:
   pull_request:
     branches:
       - release-candidate
-      - master
```

Comment:
Is this change part of the PR ?

Reply:
could have made in another PR, but pull-request-ci do not work well on pr rc -> ci due to [skip ci] and, we need to fix it by hand anyway

Reply:
I splitted history in a revert commit + the doxy one.